README.md

```diff
@@ -133,11 +138,17 @@ Open anaconda prompt, use `cd` to navigate to the directory where your local fil
 conda update conda
 conda env create -f environment.yml
 ```
-and activate the environment:
+Installation requires that the conda channel-priority be set to "flexible". This is the default behavior, so if you've never manually changed this, you shouldn't have to worry about this. However, if you receive an error message like "Found conflicts!" when trying to install the environment, try setting your channel priority to flexible by running the following command:`conda config --set channel_priority flexible` and then re-running the above commands.
+
+The final step is to install the `oge` package itself in the conda environment. To do so, run:
 ```bash
 conda activate open_grid_emissions
+pip install build
+python -m build
+pip install .
```

Comment:
I guess for the development setup we should recommend doing `pip install --editable .`. This is what we suggest doing below for `pipenv`

Reply:
Good catch! I'll update this.